### ISSUE TITLE
[Multi-tenant] Clean up firestore requests to `clientUpdates`

### ISSUE THREAD

**Issue by daschi** [Recidiviz/pulse-dashboard#2108]
**What needs to be done? Why does it need to be done?**
After we migrate the frontend to use `clientUpdatesV2` fully, we should remove the code that makes requests to `clientUpdates` as a fallback.

**Additional context**
Add any other context about the task here.

**Comment by macfarlandian** [comment]
added an additional blocker because we have a multistage migration now for this collection and it's probably easier to clean it all up once rather than incrementally? (if that turns out to not be true we can figure that out later but just flagging the change for now)

**Comment by daschi** [comment]
This seems like it can be closed now right? @macfarlandian

**Comment by macfarlandian** [comment]
I don't think we have actually removed the code? the incremental migration is still ongoing

**Comment by macfarlandian** [comment]
(I interpret this ticket to apply to the entire migration from `clientUpdates` to `clientUpdatesV2` to the subcollections, but if other tickets supersede this to cover all that then never mind)
